src/test/regress/expected/multi_agg_distinct.out

```diff
@@ -154,61 +156,61 @@ CREATE TABLE lineitem_hash (
 	l_comment varchar(44) not null );
 SET citus.shard_replication_factor TO 1;
 SELECT create_distributed_table('lineitem_hash', 'l_orderkey', 'hash');
- create_distributed_table 
---------------------------
- 
+ create_distributed_table
+---------------------------------------------------------------------
+
 (1 row)
 
-\copy lineitem_hash FROM '@abs_srcdir@/data/lineitem.1.data' with delimiter '|'
-\copy lineitem_hash FROM '@abs_srcdir@/data/lineitem.2.data' with delimiter '|'
+COPY lineitem_hash FROM :'lineitem_1_data_file' with delimiter '|';
```

Comment:
Do we have any `\copy` tests remaining in the PR ? I guess you can still do `\copy` from program, and it seems useful to have at least one such test on distributed tables?

Reply:
Several examples:

`multi_copy`

```sql
-- Test client-side copy from file
-- \copy does not support variable interpolation. Hence we store and execute
-- the query in two steps for interpolation to kick in.
-- See https://stackoverflow.com/a/67642094/4576416 for details.
\set client_side_copy_command '\\copy customer_copy_hash FROM ' :'customer3datafile' ' WITH (DELIMITER '''|''');'
:client_side_copy_command
```

-----

`pg_dump` contains 4 `\copy .. FROM PROGRAM`
```sql
\COPY output FROM PROGRAM 'pg_dump --quote-all-identifiers -h localhost -p 57636 -U postgres -d regression -t dumper.data --data-only | psql -tAX -h localhost -p 57636 -U postgres -d regression'
```

-----

`multi_multiuser_load_data` that checks various user scenarios. `COPY` needs superuser privileges to read from files on disk, and hence we use `\copy` there instead.

```sql
-- as user with no access, should fail
SET ROLE no_access;
\set copy_command '\\COPY lineitem FROM ' :'lineitem_2_data_file' ' with delimiter '''|''';'
:copy_command
RESET ROLE;
```

Reply:
After another check, I realized that the only `\copy .. FROM PROGRAM` tests write to local tables. Let me update an existing `COPY` test that uses a PROGRAM clause into `\copy`.